spec/system/task_spec.rb

```diff
@@ -2,23 +2,25 @@
 
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
+    let(:project) { FactoryBot.create(:project) }
+    let(:task) { FactoryBot.create(:task) }
```

Comment:
今回のテストケースでは、projectは1度のみ作成すれば十分なのでtaskのデータ作成時に併せて作成させるようにしましょう！

また、FactoryBotの指定は省略するのが良いです。
```suggestion
    let(:task) { create(:task) }
```